### DISCUSSION TITLE
Scene Builder with FXGL?

### DISCUSSION THREAD

**Discussion by dhickel** [AlmasB/FXGL#1158] [Q&A] [answered]
Is it possible to use Scene Builder with FXGL?

I am working on a fairly static turn-based game involving cards and could benefit from some of the FXGL features, but since the game will be heavily UI depended would like to still use scene builder to build most of the UI. A canvas would only be needed for showing the sprites, their attack animations and cards in play.

**Comment by AlmasB** [comment, marked as answer]
Hi, yes it should be fine. Scene Builder generates .fxml, which you can load with `getAssetLoader().loadUI(...)`. Everything else is pretty much the same as javafx.
